dynawo/sources/Simulation/DYNSimulationLauncher.cpp

```diff
@@ -120,6 +120,7 @@ void launchSimu(const std::string& jobsFileName) {
       // Needed as otherwise terminate might crash due to missing staticRef variables
       if (err.key() == DYN::KeyError_t::StateVariableNoReference) {
         simulation->activateExportIIDM(false);
+        simulation->setLostEquipmentsExportMode();
```

Comment:
I would prefer to have an explicit argument here instead of a default value

Reply:
Done